site/en/docs/web-platform/navigation-api/index.md

```diff
@@ -96,116 +92,96 @@ links.forEach(link => link.addEventListener('click', updatePage));
 This is fine, but not exhaustive.
 Links might come and go on your page, and they're not the only way users can navigate through pages.
 E.g., they may submit a form or even use an [image map].
-Your page might deal with these, but there's a long tail of possibilities which could just be simplified—something that the new App History API achieves.
+Your page might deal with these, but there's a long tail of possibilities which could just be simplified—something that the new Navigation API achieves.
 
 Personally, the History API often _feels_ like it could go some way to help with these possibilities.
 However, it really only has two surface areas: responding if the user presses Back or Forward in their browser, plus pushing and replacing URLs.
 It doesn't have an analogy to "navigate", except if you manually set up listeners for, e.g., click events, as demonstrated above.
 
 ## Transition
 
-When your code calls `transitionWhile()` from within its "navigate" handler, it informs the browser that it's now preparing the page for the new, updated state; and that the navigation may take some time (because the call accepts a `Promise`).
+When your code calls `transitionWhile()` from within its "navigate" listener, it informs the browser that it's now preparing the page for the new, updated state; and that the navigation may take some time. The `Promise` you pass to `transitionWhile()` tells the browser how long the navigation takes.
 
-As such, this API introduces a semantic concept that the browser understands: a SPA navigation is currently occurring, over time, changing the document from a previous URL and state to a new one.
-This has a number of potential benefits, including that of accessibility: browsers can surface the beginning, end, or potential failure of a navigation in a variety of ways.
-Chrome, for example, intends to [reactivate its native loading indicator][loading-crbug] during this time.
+As such, this API introduces a semantic concept that the browser understands: an SPA navigation is currently occurring, over time, changing the document from a previous URL and state to a new one.
+This has a number of potential benefits, including accessibility: browsers can surface the beginning, end, or potential failure of a navigation.
+Chrome, for example, activates its native loading indicator, and allows the user to interact with the stop button. (This doesn't currently happen when the user navigates via the back/forward buttons, but that (will be fixed soon)[loading-crbug].)
```

Comment:
This link syntax looks suspicious?

Reply:
Good catch (I don't like these footnote style links, so I'm not used to using them)